### PR TITLE
Fix prettier warnings in sfDivider

### PR DESCRIPTION
**Pull request by fvillata**
# Related issue
#214

# Scope of work
Fix prettier warnings. 

# Checklist

- [x] I followed [composition rules](https://github.com/DivanteLtd/storefront-ui/blob/master/docs/component-rules.md) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/atoms/SfDivider/SfDivider.stories.js [hunk 1]
```diff
@@ -22,11 +22,11 @@ storiesOf("Atoms|Divider", module)
       template: `<SfDivider />`
     }),
     {
-     info: {
-       summary: `<p>Divider element.</p>
+      info: {
+        summary: `<p>Divider element.</p>
        <h2>Usage</h2>
        <pre><code>import SfDivider from "@storefrontui/vue/dist/SfDivider.vue"</code></pre>
```
Comment (filrak) [inline packages/vue/src/components/atoms/SfDivider/SfDivider.stories.js]:
```suggestion
       <pre><code>import { SfDivider } from "@storefrontui/vue"</code></pre>
```

Comment (filrak) [reply packages/vue/src/components/atoms/SfDivider/SfDivider.stories.js]: I must've missed it previously

Comment (fvillata) [reply packages/vue/src/components/atoms/SfDivider/SfDivider.stories.js]: uh, actually I edited it in this PR I just done:
https://github.com/DivanteLtd/storefront-ui/pull/216

**Comment by filrak** [comment]
Thank you! Merging 
image

**Pull request merged**